packages/core/lib/commands/help/meta.js

```diff
@@ -1,9 +1,16 @@
+const OS = require("os");
+
 module.exports = {
   command: "help",
   description:
     "List all commands or provide information about a specific command",
   help: {
-    usage: "truffle help [<command>]",
+    usage:
+      "truffle help [<command> [<subCommand>]]" +
+      OS.EOL +
```

Comment:
Get rid of these last two usage examples as they are not correct. This file is for the help command: do note that that is a completely different thing than the `--help` flag. You understand?

Reply:
ah ok. That makes sense.